File: translation_output/misleading_test/gpt-3.5/codenet/python/java/p02771.java
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int c = scanner.nextInt();

        if (a == b && b != c) {
            System.out.println("Ye");
        } else if (b == c && a != b) {
            System.out.println("Ye");
        } else if (a == c && a != b) {
            System.out.println("Ye");
        } else {
            System.out.println("No");
        }
    }
}
```

File: translation_output/misleading_test/magicoder/codenet/java/python/p02771.py
<｜begin▁of▁sentence｜>You are an exceptionally intelligent coding assistant that consistently delivers accurate and reliable responses to user instructions.

@@ Instruction
Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
5 7 5

Expected output:
Yes9

```

import java.util.Scanner;

public class p02771 {

	public static void main(String[] args) {

		Scanner scan = new Scanner(System.in);
		int A = scan.nextInt();
		int B = scan.nextInt();
		int C = scan.nextInt();

		if((A == B && B == C)||(A!=B && A!=C && B!=C) ) {
			System.out.println("No");
		}else {
			System.out.println("Yes");
		}
	}

}

```

@@ Response
```python
# Python code

# Read input from user
A, B, C = map(int, input().split())

# Check if all numbers are equal or not
if A == B == C:
    print("No")
else:
    print("Yes")
```

Test input:
5 7 5

Expected output:
No<｜end▁of▁sentence｜>
